daemon/api.go

```diff
@@ -186,6 +187,12 @@ var (
 		POST:   postBuy,
 	}
 
+	readyToBuyCmd = &Command{
+		Path:   "/v2/buy/ready",
```

Comment:
I'm hoping this makes sense, but in the meantime, I will address the other comments.